.github/workflows/ci.yaml

```diff
@@ -36,11 +36,20 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
+    - name: install requirements
+      run: npm install
     - name: build docker
       run: docker-compose build
     - name: run crawl
-      run: docker-compose run crawler crawl --url http://www.example.com/ --generateWACZ  --collection wr-net --workers 2              
+      run: docker-compose run crawler crawl --url http://www.example.com/ --generateWACZ  --text --collection wr-net --workers 2              
     - name: validate existing wacz
       run: docker-compose run crawler wacz validate --file collections/wr-net/wr-net.wacz
+    - name: unzip wacz
+      run: docker-compose run crawler unzip collections/wr-net/wr-net.wacz -d collections/wr-net/wacz
```

Comment:
I guess could also unzip the `unzip collections/wr-net/wr-net.wacz pages/pages.jsonl` here, which would then fail if it didn't exist, saving the extra check later. But, this makes it easier to add additional tests if we want to later on other files.

Reply:
yeah i figured we would be adding more tests as we add to development